### DISCUSSION TITLE
Are there any plans to include more antd block in the future?

### DISCUSSION THREAD

**Discussion by kemeris2000** [lowdefy/lowdefy#1814] [Ideas and feature requests]
For example new [Step](https://ant.design/components/steps) block

![image](https://github.com/lowdefy/lowdefy/assets/48207709/7299d968-ebce-4dae-843a-b845e1c3790e)

update [Statistic](https://ant.design/components/statistic#components-statistic-demo-countdown) block to support Countdown

![image](https://github.com/lowdefy/lowdefy/assets/48207709/724b4937-3eba-413d-bd08-e8065e60a38d)

**Comment by Gervwyk** [comment]
Hi @kemeris2000 

Always keen to add more Antd blocks and PRs are welcome. Dispay type blocks the easiest ones since it's often just a map of props to the Antd component. Would be happy to accept a PR. Have a look at this folder if you want to give it a try, I can provide some assistance: 
https://github.com/lowdefy/lowdefy/tree/main/packages/plugins/blocks/blocks-antd/src/blocks

Altough these blocks are not on our radar currently, I see if we can find some time to include next, their APIs are easy.

**Comment by kemeris2000** [comment]
Good, I will try to look at Title block example.
